explore/gamma-hurdle/parameter_estimates.ipynb

```diff
@@ -0,0 +1,169 @@
+      "text/plain": [
+       "<Figure size 432x288 with 1 Axes>"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "plt.scatter(df['b_mle'], df['b_match'])\n",
```

Comment:
Do you think the increase in our estimator's variance as beta gets larger will negatively affect our model? Do we have reason to believe that when beta gets into the 20-40 range in the database should we switch to the MLE estimator? Or should they be about the same accuracy-wise?

Reply:
The MLE estimate here is approximate, while the method of moments estimate is exact. I don't think estimation quality (various [advantages](https://en.wikipedia.org/wiki/Method_of_moments_(statistics)#Advantages_and_disadvantages) of MLE over method of moments) isn't a major concern, since the quasi p-values computed are not actual hypothesis tests, just a way to determine which DWPC values are more or less likely to have occurred due to degree alone.